packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -289,7 +300,8 @@ type AccountInternalProps = {
   categoryGroups: ReturnType<typeof useCategories>['grouped'];
   hideFraction: boolean;
   accountsSyncing: string[];
-} & ReturnType<typeof useActions>;
```

Comment:
Remove the useActions prop here. Instead we will pass in the dispatch function and dispatch the actions manually instead.